recipes/protoc/3.9.x/conanfile.py

```diff
@@ -50,37 +59,25 @@ def build(self):
 
     def package(self):
         self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
+        # os.makedirs(os.path.join(self.package_folder, self._cmake_base_path))
         cmake = self._configure_cmake()
         cmake.install()
-        cmake_folder = os.path.join(self.package_folder, self._cmake_base_path)
-        os.unlink(os.path.join(cmake_folder, "protoc-config-version.cmake"))
-        # FIXME: To find protoc the cmake generator is required
-        cmake_path = os.path.join(cmake_folder, "protoc-targets-noconfig.cmake")
-        tools.replace_in_file(cmake_path, "${_IMPORT_PREFIX}/bin/", "${CONAN_BIN_DIRS_PROTOC}/")
 
     def package_id(self):
         del self.info.settings.compiler
-        del self.info.settings.arch
         self.info.include_build_settings()
 
     def package_info(self):
         bindir = os.path.join(self.package_folder, "bin")
         self.output.info("Appending PATH environment variable: {}".format(bindir))
         self.env_info.PATH.append(bindir)
-        self.cpp_info.builddirs = [self._cmake_base_path]
-        # INFO: Google Protoc exports a bunch of functions/macro that can be consumed by CMake
-        # protoc-config.cmake: provides protobuf_generate function
-        # protoc-module.cmake: provides legacy functions, PROTOBUF_GENERATE_CPP PROTOBUF_GENERATE_PYTHON
-        # protoc-options.cmake: required by protoc-tools.cmake
-        # protoc-targets.cmake: required by protoc-tools.cmake
-        # protoc-targets-noconfig.cmake: declare protobuf:protoc as target
+
         self.cpp_info.build_modules = [
-            os.path.join(self._cmake_base_path, "protoc-config.cmake"),
             os.path.join(self._cmake_base_path, "protoc-module.cmake"),
             os.path.join(self._cmake_base_path, "protoc-options.cmake"),
-            os.path.join(self._cmake_base_path, "protoc-targets.cmake"),
-            os.path.join(self._cmake_base_path, "protoc-targets-noconfig.cmake")
+            os.path.join(self._cmake_base_path, "protoc-generate.cmake"),
```

Comment:
what is that protoc-generate.cmake ?

Reply:
It's the `protobuf-config.cmake` renamed and with all package finding stuff disabled.

Reply:
sounds pretty interesting :D

Reply:
I named it generate because it contains the `protobuf_generate_cpp` functions.